queries.sql

```diff
@@ -6,3 +6,43 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Update species to unspecified
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+-- Classify species into digimon and pokemon 
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
```

Comment:
- [ ] Based on the former reviewer's comment, you ought to add screenshots for the following transactions that you wrote:

```SQL
BEGIN TRANSACTION;
UPDATE animals SET species = 'unspecified';
ROLLBACK;

BEGIN TRANSACTION;
UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
COMMIT;
```

#### Hint: After committing or rolling back the transactions, you can run this command:

`SELECT * from animals;`, then add the screenshot to your queries_image directory. 👍 😎

Reply:
@Gambit142 Thanks a lot for the feedback and tips. It is much clearer now.

Reply:
@ezraermy, you are welcome. 👍